django-stubs/contrib/gis/geoip2.pyi

```diff
@@ -0,0 +1,75 @@
+from collections.abc import Sequence
+from ipaddress import IPv4Address, IPv6Address
+from pathlib import Path
+from typing import Final, TypedDict
+
+from django.contrib.gis.geos import Point
+from django.utils.functional import cached_property
+from typing_extensions import Self, TypeAlias, deprecated
+
+HAS_GEOIP2: bool
+SUPPORTED_DATABASE_TYPES: set[str]
+
+_CoordType: TypeAlias = tuple[float, float] | tuple[None, None]
+_QueryType: TypeAlias = str | IPv4Address | IPv6Address
+
+class _CityResponse(TypedDict):
+    accuracy_radius: int | None
+    city: str
+    continent_code: str
+    continent_name: str
+    country_code: str
+    country_name: str
+    is_in_european_union: bool
+    latitude: float
+    longitude: float
+    metro_code: int | None
+    postal_code: str | None
+    region_code: str | None
+    region_name: str | None
+    time_zone: str | None
+    # Kept for backward compatibility.
+    dma_code: int | None
+    region: str | None
+
+class _CountryResponse(TypedDict):
+    continent_code: str
+    continent_name: str
+    country_code: str
+    country_name: str
+    is_in_european_union: bool
+
+class GeoIP2Exception(Exception): ...
+
+class GeoIP2:
+    MODE_AUTO: Final[int]
```

Comment:
Do we care about actual values here? Or just `int` is good enough?

Reply:
I tried `Final[Literal[0]]` but the linter wanted that changed to `Final` only 🤔

Unless we should do `MODE_AUTO: Final = 0`, etc?

Reply:
We can use `=`, sure :)